service/src/test/java/com/github/swapbook/repositories/UsersRepositoryTest.java

```diff
@@ -0,0 +1,26 @@
+package com.github.swapbook.repositories;
+
+import com.github.swapbook.model.User;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UsersRepositoryTest {
+
+
+    @Test
+    public void testAdd(){
```

Comment:
Jak tu jest z podejściem AAA? 
https://dariuszwozniak.net/posts/kurs-tdd-3-struktura-test-czyli-arrange-act-assert

Reply:
racja, podzieliłem ci to trochę, by było czytelniej